### DISCUSSION TITLE
git command

### DISCUSSION THREAD

**Discussion by jenilgajera** [community/community#137790] [Other Feature Feedback, Questions, & Ideas] [answered]
is  it important to use git command can we use manually ?

**Comment by nenisrudani7** [comment, marked as answer]
no , you can commit using github desktop.

**Comment by WazedKhan** [comment]
@jenilgajera No, it's not strictly necessary to use Git commands. Tools like GitHub Desktop(mentioned by @nenisrudani7) and modern IDEs such as VS Code, PyCharm, etc. provide UI for it.
